sdk/iot/Azure.Iot.Hub.Service/src/Authentication/IotHubSasCredential.cs

```diff
@@ -144,11 +143,23 @@ private AccessToken GetSasTokenAsync(CancellationToken cancellationToken)
             }
         }
 
+        /// <summary>
+        /// Gets an Azure.Core.AccessToken for the specified set of scopes.
```

Comment:
`<see cref="Azure.Core.AccessToken"/>`

Reply:
I will add it here. But we need a task to update almost all comments and add goodness to our C# track 2 SDK.